apps/aecore/lib/aecore/keys/worker.ex

```diff
@@ -0,0 +1,359 @@
+defmodule Aecore.Keys.Worker do
```

Comment:
why is this added back?

Reply:
@gspasov shouldn't the wallet be used?

Reply:
It has slipped through us for some reason